File: Logistics.Web/ClientApp/src/app/models/item-decommission.ts
import { Item } from './item';
import { ItemDecommissionVerification } from './item-decommission-verification';
import { User } from './user';

export class ItemDecommission {
  id: number;
  itemId: number;
  userId: number;
  remarks: string;
  isVerified: boolean;
  decommissionDate: Date;
  item: Item;
  itemDecommissionVerification: ItemDecommissionVerification;
  user: User;
}

File: Logistics.Data/ItemDecommission.cs
﻿using System;

namespace Logistics.Data
{
    public class ItemDecommission
    {
        public int Id { get; set; }
        public int ItemId { get; set; }
        public int UserId { get; set; }
        public string Remarks { get; set; }
        public bool IsVerified { get; set; }
        public DateTime DecommissionDate { get; set; }

        public Item Item { get; set; }
        public ItemDecommissionVerification ItemDecommissionVerification { get; set; }
        public User User { get; set; }
    }
}
